src/plugins/azureBasePlugin.ts

```diff
@@ -10,6 +10,10 @@ export abstract class AzureBasePlugin<TOptions=Serverless.Options> {
   protected config: ServerlessAzureConfig;
   protected commands: ServerlessCommandMap;
   protected processedCommands: ServerlessCliCommand[];
+  /**
+   * Specifies if targeted runtime is linux
+   */
+  protected isLinuxTarget: boolean;
```

Comment:
Doesn't feel like this property should exist on the base plugin since its a concern of deployment

Reply:
I think this will go away when I address your other feedback on reducing some of the complexity

Reply:
Add to config service